### DISCUSSION TITLE
Where does WingetUI default to installing npm packages?

### DISCUSSION THREAD

**Discussion by Quitch** [Devolutions/UniGetUI#2105] [Q&A] [answered]
I'd assumed it would install them globally, as I couldn't see how else it could meaningfully monitor and update them. However, this doesn't appear to be the case so I'm not sure where it puts them. Also, how can this be usefully combined with existing npm package locations?

**Comment by marticliment** [comment, marked as answer]
They are installed under C:\Users\<user>
